### ISSUE TITLE
docker action plugin moved but synchronize doesn't know that

### ISSUE THREAD

**Issue by SirUli** [ansible-collections/ansible.posix#132]
##### SUMMARY
Action plugin synchronize requires an update due to the new path of docker. Currently it fails [here](https://github.com/ansible-collections/ansible.posix/blob/main/plugins/action/synchronize.py#L195) when attempting to copy data to docker due to docker having moved from community.general.docker to community.docker.docker.

Looking at the overall source code of this plugin this might need a change of multiple locations (e.g. L63, 173 etc) towards the new location of the module.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
synchronize

##### ANSIBLE VERSION
```
2.10.6
```

##### CONFIGURATION
-

##### OS / ENVIRONMENT
-

##### STEPS TO REPRODUCE
Run a docker container, then run a synchronize task to sync data into this docker container.

##### EXPECTED RESULTS
Data is synced successfully

##### ACTUAL RESULTS

```
fatal: [container]: FAILED! => {"changed": false, "msg": "synchronize uses rsync to function. rsync needs to connect to the remote host via ssh, docker client or a direct filesystem copy. This remote host is being accessed via community.docker.docker instead so it cannot work."}
```

**Comment by glillico** [comment]
I am seeing the same issue and after chatting with tadeboro on the ansible-molecule irc channel, it was pointed out that the docker elements have now been moved to the community.docker collection, however the synchronize module has not been updated to include this collection and needs community.docker.docker adding.

**Comment by rockandska** [comment]
Just hit the same error with `ansible 2.9.27`, not sure it was backported

**Comment by tadeboro** [comment]
> Just hit the same error with `ansible 2.9.27`, not sure it was backported

The 2.9 Ansible release only receives security fixes, so this will not be backported. What you can do is install `ansible.posix` collection and use FQCN in your playbook. This should fix things on 2.9.

**Comment by rockandska** [comment]
Already tried with molecule==3.4.0,molecule-docker==1.1.0
and something like
```
ansible.posix.synchronize
  src: /tmp/
  dest: /tmp/
```

But hit this error with ansible
```
ERROR! couldn't resolve module/action 'ansible.posix.synchronize'. This often indicates a misspelling, missing collection, or incorrect module path.

The error appears to be in '/home/rockandska/Documents/Git/ansible-role-rabbitmq/molecule/cluster/pre_tasks.yml': line 44, column 11, but may
be elsewhere in the file depending on the exact syntax problem.
```

Will try to update molecule / molecule-docker
